lib/Library.php

```diff
@@ -170,58 +170,56 @@ public static function FilterArtwork(string $query = null, string $status = null
 		// "unverified-submitter": Show unverified artwork from the submitter
 		// "in-use": Show only in-use artwork
 
-		$artworks = [];
+		$statusCondition = '';
+		$params = [];
 
 		if($status === null || $status == 'all'){
-			$artworks = Db::Query('
-				SELECT *
-				from Artworks
-				where Status in (?)', [ArtworkStatus::Approved->value], 'Artwork');
+			$statusCondition = 'Status = ?';
+			$params[] = ArtworkStatus::Approved->value;
 		}
 		elseif($status == 'all-admin'){
-			$artworks = Db::Query('
-				SELECT *
-				from Artworks', [], 'Artwork');
+			$statusCondition = 'true';
 		}
 		elseif($status == 'in-use'){
-			$artworks = Db::Query('
-				SELECT *
-				from Artworks
-				where EbookWwwFilesystemPath is not null
-				', [], 'Artwork');
+			$statusCondition = 'EbookWwwFilesystemPath is not null';
```

Comment:
This matches the previous logic but I think there was an error in the previous logic. We should also check that the status is 'approved' so we don't accidentally show declined artwork that has been marked as in use.

Reply:
You're right. I confirmed that was a problem before and fixed it here.

If you think this PR is good to go, you can squash these 3 little commits and merge it. I will have a whack at the upgraded search functionality soon.